cmake/fmtlib.cmake

```diff
@@ -0,0 +1,31 @@
+include(ExternalProject)
```

Comment:
Do you actually use anything from `ExternalProject` here? I only see `FetchContent` and that has a separate include.

Reply:
you are right. that's a leftover. I first feared that somwhere we're on a very old cmake where we don't have FetchContent. will drop this line.